Tools/ardupilotwaf/boards.py

```diff
@@ -17,6 +17,9 @@ def __init__(cls, name, bases, dct):
         if cls.abstract:
```

Comment:
> So, that's where I think wasn't clear enough. My idea was that, in Board'sconfigure() function, we set env.TOOLCHAIN = self.toolchain just before loading compiler_{c,cxx}. If we don't do this, all places that need to know which toolchain is in use need to import board, which is an unnecessary dependency IMO. And I think the less tools depend on each other, the better.

Ok, I'll do that. I was thinking that you didn't want that just because that's setting a var in env and after that we call configure_env, which can seem weird. But I agree that's preferable to always have to import board when we need to know the toolchain.